WordPress/Classes/ViewRelated/Menus/Views/MenusHeaderView.m

```diff
@@ -0,0 +1,151 @@
+#import "MenusHeaderView.h"
+#import "MenusSelectionView.h"
+#import "Blog.h"
+#import "WPStyleGuide.h"
+#import "Menu.h"
+#import "Menu+ViewDesign.h"
+#import "MenuLocation.h"
+#import "WPFontManager.h"
+
+@interface MenusHeaderView () <MenusSelectionViewDelegate>
+
+@property (nonatomic, weak) IBOutlet UIStackView *stackView;
+@property (nonatomic, strong, readonly) Blog *blog;
+@property (nonatomic, weak) IBOutlet MenusSelectionView *locationsView;
+@property (nonatomic, weak) IBOutlet MenusSelectionView *menusView;
+@property (nonatomic, weak) IBOutlet UILabel *textLabel;
+
+@end
+
+@implementation MenusHeaderView
+
+- (void)awakeFromNib
+{
+    [super awakeFromNib];
+
+    self.stackView.spacing = MenusDesignDefaultContentSpacing / 2.0;
+    
+    self.backgroundColor = [WPStyleGuide greyLighten30];
+    
+    self.locationsView.delegate = self;
+    self.menusView.delegate = self;
+    
+    self.textLabel.font = [WPFontManager systemRegularFontOfSize:13];
+    self.textLabel.backgroundColor = [UIColor clearColor];
+    self.textLabel.textColor = [WPStyleGuide greyDarken20];
+    self.textLabel.text = NSLocalizedString(@"USES", @"Menus label for describing which menu the location uses in the header.");
+}
+
+- (void)setupWithMenusForBlog:(Blog *)blog
+{
+    _blog = blog;
```

Comment:
Since this is not a setter or `init` method, using the setter method for the property would be preferred (ie: `self.blog = blog`).